src/store/selectors.ts

```diff
@@ -296,7 +297,8 @@ export const selectTransferTargetError = (state: RootState) => {
     state.transfer.relayerFee === undefined &&
     // Acala offers relaying without a fee for qualified tokens
     state.transfer.targetChain !== CHAIN_ID_ACALA &&
-    state.transfer.targetChain !== CHAIN_ID_KARURA
+    state.transfer.targetChain !== CHAIN_ID_KARURA &&
+    state.transfer.targetChain !== CHAIN_ID_NEON
```

Comment:
We should handle neon in the case below this one too

Reply:
I added requirements to below case